packages/internals/src/utils/path.ts

```diff
@@ -0,0 +1,15 @@
+import path from 'path'
+
+/**
+ * Normalize `filePath` to use forward slashes as a separator. `filePath` is
+ * treated as a path specific to the current platform, so backslashes will only
+ * be replaced with forward slashes on Windows. On other platforms, where a
+ * backslash is a valid filename character, it will be treated as such and will
+ * not be replaced.
+ */
+export function pathToPosix(filePath: string): string {
+  if (path.sep === path.posix.sep) {
+    return filePath
+  }
+  return filePath.split(path.sep).join(path.posix.sep)
```

Comment:
So Windows->Unix will now work, will Unix->Windows work?

Reply:
Windows supports both `/` and `\`, it just uses `\` by convention. Normalizing everything to `/` works on both Unix and Windows.